nano/node/json_handler.cpp

```diff
@@ -1679,6 +1680,8 @@ void nano::json_handler::bootstrap_status ()
 		response_l.put ("target_connections", std::to_string (attempt->target_connections (attempt->pulls.size ())));
 		response_l.put ("total_blocks", std::to_string (attempt->total_blocks));
 		response_l.put ("runs_count", std::to_string (attempt->runs_count));
+		response_l.put ("requeued_pulls", std::to_string (attempt->requeued_pulls));
+		response_l.put ("confirmed_frontiers", std::to_string (attempt->confirmed_frontiers));
```

Comment:
Renaming this to `frontiers_confirmed` makes it more clear it's a boolean

Reply:
Updated